elm/lib/dependabot/elm/file_parser.rb

```diff
@@ -149,7 +162,8 @@ def build_elm_json_dependency(name:, group:, requirement:, direct:)
 
       sig { returns(String) }
       def repo_type
-        parsed_elm_json.fetch("type")
+        type = parsed_elm_json.fetch("type")
+        T.must(type.is_a?(String) ? type : nil)
```

Comment:
Not sure what this is doing but I think the result is nilable so I don't think the signature should not be `sig { returns(String) }`